tool/colors/generate_files.dart

```diff
@@ -43,15 +43,24 @@ Future<void> exitWith(int code) async {
 }
 
 void generatePropertiesFiles() {
-  generateProperties(material.colors, '$outputFolder/material.properties');
-  generateProperties(cupertino.colors, '$outputFolder/cupertino.properties');
-  generateProperties(css.colors, '$outputFolder/css.properties');
+  generateProperties(
+    material.colors,
+    path.join(outputFolder, 'material.properties'),
+  );
+  generateProperties(
+    cupertino.colors,
+    path.join(outputFolder, 'cupertino.properties'),
+  );
+  generateProperties(
+    css.colors,
+    path.join(outputFolder, 'css.properties'),
+  );
```

Comment:
Maybe we could simplify all of these by prepending `outputFolder` inside `generateProperties`/`generateJson` and have just the filenames here?

Reply:
Sure, sounds good.